<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesForm</class>
 <widget class="QWidget" name="NotesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetButtons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonBold">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/f.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonItalic">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/k.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonUnderline">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/u.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLarger">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/bigger.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSmaller">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/smaller.png</normalon>
            </iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonColFG">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/fore.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonColBG">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/back.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonImage">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/QTTestVs10/image.png</normalon>
            </iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="margin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonTable">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normalon>:/QTTestVs10/table.png</normalon>
               </iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Zeilen:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxRows">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Spalten:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxCols">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCode">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/QTTestVs10/code.png</normalon>
         </iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>596</width>
        <height>355</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEditNotes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qttestvs10.qrc"/>
 </resources>
 <connections/>
</ui>
